docs/_newsfragments/1589.breakingchange.rst

```diff
@@ -0,0 +1,3 @@
+Deserializing an empty JSON body on a request will return a BadRequest.
+Previously the ``None`` was returned. To retain the current behaviour
```

Comment:
Probably you meant: :arrow_right: _To retain the previous behavior, please <...>_?